lib/main.dart

```diff
@@ -194,78 +196,79 @@ class _MyHomePageState extends State<MyHomePage> {
-                          // digits plus dot
-                          FilteringTextInputFormatter.allow(RegExp(r'[.0-9]'))
-                        ],
-                      ),
+          width: 1200,
+          child: Column(
+            children: [
+              const Row(
+                children: [
+                  Expanded(child: RpcWidget()),
+                ],
+              ),
+              const Row(
+                children: [Text('Withdrawal stuff')],
+              ),
+              Row(
+                children: [
+                  Text('Withdrawal bundle status: $_withdrawalBundleStatus'),
+                ],
+              ),
+              Row(
+                children: [
+                  // TODO: this needs to be a P2PKH address. Validate this,
+                  // somehow.
+                  // can probably use https://github.com/dart-bitcoin/bitcoin_flutter
+                  // addressToOutputScript
+                  const Text('Mainchain address: '),
+                  Expanded(
+                    child: TextField(
+                      controller: _withdrawalAddress,
                     ),
-                  ],
-                ),
-                Row(
-                  children: [
-                    ElevatedButton(
-                      onPressed: _onWithdraw,
-                      child: const Text("Submit withdrawal"),
-                    )
-                  ],
-                ),
-                const Withdrawals(),
-                Row(
-                  children: const [Text("Deposit stuff")],
-                ),
-                Row(
-                  children: [
-                    DepositAddress(_depositAddress),
-                    ElevatedButton(
-                      onPressed: () async {
-                        var address = await generateDepositAddress();
-                        setState(() {
-                          _depositAddress = address;
-                        });
-                      },
-                      child: const Text("Generate"),
+                  ),
+                ],
+              ),
+              Row(
+                children: [
+                  const Text('Amount: '),
+                  Expanded(
+                    child: TextField(
+                      controller: _withdrawalAmount,
+                      inputFormatters: [
+                        // digits plus dot
+                        FilteringTextInputFormatter.allow(RegExp(r'[.0-9]')),
+                      ],
                     ),
-                  ],
-                )
-              ],
-            )),
+                  ),
+                ],
+              ),
+              Row(
+                children: [
+                  ElevatedButton(
+                    onPressed: _onWithdraw,
+                    child: const Text('Submit withdrawal'),
+                  ),
+                ],
+              ),
+              const Withdrawals(),
+              const Row(
```

Comment:
could we do formating changes in a single commit, preferably in the same commit that the spec for formating is changed?